### DISCUSSION TITLE
Flatpak

### DISCUSSION THREAD

**Discussion by D8ma** [linuxdeepin/developer-center#3284] [Features Request & Ideas | 特性请求 & 头脑风暴]
Will flatpak work as in elementary OS? Dt distributions need to use flatpak,AppImage and the app store (MintInstall,AppCenter,KDE Discover,GNOME Software)

**Comment by BLumia** [comment]
It will work but probably not as integrated in the App Store app as other distro did.

btw, please edit your discussion title so it could be clear so others could get a brief idea about what's the theme of the discussion thread.
